Documentation/GettingStarted/Solr.rst

```diff
@@ -67,24 +67,25 @@ You should see the web interface of Solr to run queries:
 
 |
 
+**Important**: The image "typo3-solr" ships a default core for all languages. The data of the cores is stored on a data volume. When you want to update the container, you can just start a new container using the data volume of the old container. But at the same time this has the limitation, that you should only use this image with the default cores! When you want to create custom cores with a different configuration please read the section "Advanced docker usage"
```

Comment:
"Advanced **D**ocker **U**sage"  

* Uppercase "D" in Docker since you're referring to it as the product/name, not the command line tool
* Uppercase "U" in title-case - http://www.grammar-monster.com/lessons/capital_letters_title_case.htm